### PR TITLE
General fixes in tutorials

### PR DESCRIPTION
**Pull request by michalbundyra**
Changes:
- aligned indents
- short array syntax
- `charset=utf8` in `dsn` configuration (database)
- trailing comma in arrays
- filters and validators changed to `::class`
- `Int` validator changed to `ToInt`
- routing types changed to `::class`

### CODE REVIEW

**Comment by froschdesign** [comment]
Please add always a meaningful title and a small description to your PRs. Thanks!

File: doc/book/db-adapter.md [hunk 1]
```diff
@@ -72,8 +72,8 @@ use Zend\Db\TableGateway\Feature\GlobalAdapterFeature;
 
 return [
     'db' => [
-        'driver'         => 'Pdo',
-        'dsn'            => 'mysql:dbname=zf2tutorial;host=localhost',
+        'driver' => 'Pdo',
```
Comment (froschdesign) [inline doc/book/db-adapter.md]: The alignment was good. Please revert.

File: doc/book/getting-started-with-zend-studio/the-application.md [hunk 2]
```diff
@@ -549,7 +548,7 @@ return [
         'driver' => 'Pdo',
         'dsn' => 'mysql:dbname=mytasklist;hostname=localhost',
         'driver_options' => [
-            PDO::MYSQL_ATTR_INIT_COMMAND => 'SET NAMES \'UTF8\''
+            PDO::MYSQL_ATTR_INIT_COMMAND => 'SET NAMES \'UTF8\'',
```
Comment (froschdesign) [inline doc/book/getting-started-with-zend-studio/the-application.md]: Please remove this. [It's no longer needed, use `charset` in `dsn` instead.](http://php.net/manual/en/ref.pdo-mysql.connection.php#refsect1-ref.pdo-mysql.connection-changelog)

File: doc/book/getting-started/routing-and-controllers.md [hunk 3]
```diff
@@ -49,9 +50,9 @@ return [
     'router' => [
         'routes' => [
             'album' => [
-                'type'    => 'segment',
+                'type'    => Segment::class,
                 'options' => [
-                    'route'    => '/album[/:action[/:id]]',
+                    'route' => '/album[/:action[/:id]]',
```
Comment (froschdesign) [inline doc/book/getting-started/routing-and-controllers.md:55]: The alignment was good. Please revert.

File: doc/book/in-depth-guide/data-binding.md [hunk 4]
```diff
@@ -189,24 +189,26 @@ Update the 'blog' `child_routes` to add the new route:
 ```php
 // In module/Blog/config/module.config.php:
 
+use Zend\Router\Http\Segment;
+
 return [
     'service_manager' => [ /* ... */ ],
     'controllers'     => [ /* ... */ ],
-    'router'          => [
+    'router' => [
```
Comment (froschdesign) [inline doc/book/in-depth-guide/data-binding.md]: The alignment was good. Please revert.

File: doc/book/in-depth-guide/data-binding.md [hunk 5]
```diff
@@ -217,7 +219,7 @@ return [
             ],
         ],
     ],
-    'view_manager'    => [ /* ... */ ],
+    'view_manager' => [ /* ... */ ],
```
Comment (froschdesign) [inline doc/book/in-depth-guide/data-binding.md]: Same here.

File: doc/book/in-depth-guide/first-module.md [hunk 6]
```diff
@@ -158,11 +160,11 @@ return [
             // Define a new route called "blog"
             'blog' => [
                 // Define a "literal" route type:
-                'type' => 'literal',
+                'type' => Literal::class,
                 // Configure the route itself
                 'options' => [
                     // Listen to "/blog" as uri:
-                    'route'    => '/blog',
+                    'route' => '/blog',
```
Comment (froschdesign) [inline doc/book/in-depth-guide/first-module.md:163]: Same here.

File: doc/book/in-depth-guide/understanding-routing.md [hunk 7]
```diff
@@ -380,20 +382,18 @@ return [
...
                     'defaults' => [
                         'controller' => Controller\ListController::class,
                         'action'     => 'index',
                     ],
                 ],
-
                 'may_terminate' => true,
-
-                'child_routes'  => [
+                'child_routes' => [
```
Comment (froschdesign) [inline doc/book/in-depth-guide/understanding-routing.md]: Same here.

File: doc/book/in-depth-guide/understanding-routing.md [hunk 8]
```diff
@@ -408,7 +408,7 @@ return [
             ],
         ],
     ],
-    'view_manager'    => [ /* ... */ ],
+    'view_manager' => [ /* ... */ ],
```
Comment (froschdesign) [inline doc/book/in-depth-guide/understanding-routing.md]: Same here.

File: doc/book/in-depth-guide/zend-form-zend-form-fieldset.md [hunk 9]
```diff
@@ -258,42 +257,42 @@ to it:
...
 
+use Zend\Router\Http\Literal;
+use Zend\Router\Http\Segment;
 use Zend\ServiceManager\Factory\InvokableFactory;
 
 return [
     'service_manager' => [ /* ... */ ],
     'controllers'     => [ /* ... */ ],
-    'router'          => [
+    'router' => [
```
Comment (froschdesign) [inline doc/book/in-depth-guide/zend-form-zend-form-fieldset.md]: Same here.

File: doc/book/in-depth-guide/zend-form-zend-form-fieldset.md [hunk 10]
```diff
@@ -258,42 +257,42 @@ to it:
...
                         'controller' => Controller\ListController::class,
                         'action'     => 'index',
-                    ]
+                    ],
                 ],
-
                 'may_terminate' => true,
-
-                'child_routes'  => [
+                'child_routes' => [
```
Comment (froschdesign) [inline doc/book/in-depth-guide/zend-form-zend-form-fieldset.md]: Same here.

File: doc/book/db-adapter.md [hunk 11]
```diff
@@ -72,8 +72,8 @@ use Zend\Db\TableGateway\Feature\GlobalAdapterFeature;
...
+        'dsn' => 'mysql:dbname=zf2tutorial;host=localhost',
```
Comment (froschdesign) [inline doc/book/db-adapter.md]: Please add `charset` to `dsn`.

File: doc/book/getting-started/routing-and-controllers.md (hunk 3 above)
Comment (michalbundyra) [inline doc/book/getting-started/routing-and-controllers.md:55]: Why it was good? It was in the same column as `defaults` but not the same as `constraints`... so maybe even these 3 values should be aligned? It is very inconsistent at all, we can always align key-value pairs, but usually it looks odd for big, multidimensional arrays.

File: doc/book/in-depth-guide/first-module.md (hunk 6 above)
Comment (michalbundyra) [inline doc/book/in-depth-guide/first-module.md:163]: again, it is inconsistent, for example why above `type` and `options` are not aligned to the same column?

**Pull request merged**